core/src/main/java/org/jdbi/v3/mapper/InferredRowMapperFactory.java

```diff
@@ -11,21 +11,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3;
+package org.jdbi.v3.mapper;
 
 import static org.jdbi.v3.Types.findGenericParameter;
 
 import java.lang.reflect.Type;
 import java.util.Optional;
 
+import org.jdbi.v3.StatementContext;
 import org.jdbi.v3.tweak.RowMapper;
 
-class InferredMapperFactory implements RowMapperFactory
+public class InferredRowMapperFactory implements RowMapperFactory
```

Comment:
If this class is going public, we probably want to add a short javadoc blurb explaining what it's for.